### DISCUSSION TITLE
Internet access required by the Grid - how to remove the need?

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#8890] [Unsupported Versions] [answered]
Hi all
We are trying to implement a distributed grid where ONLY the manager needs to have access to the internet for ALL Security Onion updates etc.  We have installed using the Direct+Manager option, but have found that the nodes (search and forward) are still going to the internet, specifically to the kolide.co site.  
Is there a way to configure the grid so only the manager needs to access the internet, and to prevent the rest of the grid going to the internet.  This would include yum updates etc.

Thanks

Ross

**Comment by dougburks** [comment, marked as answer]
One option might be to set up a proxy server and then choose the `Proxy + Manager` option:
![image](https://user-images.githubusercontent.com/1659467/195916766-dea83c36-715f-4cd8-8e25-64e9a035838f.png)
For more information about the Proxy option, please see:
https://docs.securityonion.net/en/2.3/proxy.html

Another option might be to do a full `Airgap` deployment:
![image](https://user-images.githubusercontent.com/1659467/195917285-ed6e6784-5412-42e9-a979-0012f4a367ea.png)
For more information about the Airgap option, please see:
https://docs.securityonion.net/en/2.3/airgap.html
